bql/planner/data_access_test.go

```diff
@@ -446,20 +445,19 @@ func TestDataAccessTripleToRowPredicateBindings(t *testing.T) {
 		// Setup for test:
 		tpl, err := triple.Parse(entry.t, literal.DefaultBuilder())
 		if err != nil {
-			t.Fatalf(`triple.Parse failed to parse valid triple "%s" with error: %v`, entry.t, err)
+			t.Fatalf(`triple.Parse failed for triple "%s", got error: %v`, entry.t, err)
```

Comment:
The reader already knows this is a failure/error (because the test failed). You can be a bit more concise in your message:

```
t.Fatalf(`triple.Parse failed for triple "%s": %v`, entry.t, err)
```

Reply:
Done. Thank you Tati. 🙂